src/main/kotlin/racingcar/RacingGame.kt

```diff
@@ -37,19 +49,25 @@ class InputView {
 }
 
 class ResultView {
-    fun printResult(cars: List<Car>) {
-        for (car in cars) {
-            println("-".repeat(car.position))
+    fun printResult(namesAndPositionsOfCars: List<Pair<String, Int>>) {
+        for (nameAndPosition in namesAndPositionsOfCars) {
```

Comment:
Pair를 사용하면 해당 객체가 어떠한 값을 지니고 있는지 의미를 해석하기 쉽지 않을 수 있습니다.

따라서 객체를 사용하시거나 구조분해를 이용해보시면 좋을것 같습니다.

ex. 구조분해 
```suggestion
        for ((name, position) in namesAndPositionsOfCars) {
```

Reply:
안그래도 first, second 라고 적으려니 어색했는데 구조분해를 사용하니 좋네요!